src/cli/command-registrar.ts

```diff
@@ -232,9 +233,16 @@ export default class CommandRegistrar {
     ) {
       process.stdout.write(
         chalk.yellow(
-          `warning: '${chalk.bold(subcommand)}' is not a valid command.\nsee 'bit --help' for additional information.\n`
+          ` warning: '${chalk.bold(
+            subcommand
+          )}' is not a valid command.\nsee 'bit --help' for additional information.\n`
         )
       );
+      const suggestion = didYouMean(subcommand, commander.commands.map(cmd => cmd._name));
+      if (suggestion) {
+        // @ts-ignore
```

Comment:
please remove the ts-ignore.
the issue is that didYouMean returns a string or string array (in case of multiple matches)
you can use this option: https://github.com/dcporter/didyoumean.js#returnfirstmatch to only return the first match (but you should still have to fix the ts error by checking it's not an array)
or add a condition to check if it's array and print something a bit different.

Reply:
Can you give me a better picture of the implementation strategy that you have in mind?.

Reply:
sure. 
something like:
```js
if (suggestion) {
        if (typeof suggestion === 'string') {
          console.log(chalk.red(`Did you mean '${chalk.bold(suggestion)}'?`));
        // This should never called since didYouMean.returnFirstMatch = true;
        } else {
          // option 1
          console.log(chalk.red(`Did you mean '${chalk.bold(suggestion[0])}'?`));
          // option 2 (in that case maybe don't use the didYouMean.returnFirstMatch)
          const suggestionList = suggestion.join(' or ');
          console.log(chalk.red(`Did you mean '${chalk.bold(suggestionList)}'?`));
        }
      }
```